src/extension.ts

```diff
@@ -46,17 +52,20 @@ import {
     ServerOptions,
     TransportKind,
 } from 'vscode-languageclient/node';
-import type { installFailed, showOutput } from 'vscode-webhint/dist/src/utils/notifications';
-import { activationEvents } from '../package.json';
+import { installFailed, showOutput, reloadAllProjectsConfig } from 'vscode-webhint/dist/src/utils/notifications';
 
 let telemetryReporter: Readonly<TelemetryReporter>;
 let browserInstance: Browser;
 let cdpTargetsProvider: CDPTargetsProvider;
+let globalStoragePath: string | undefined;
+
+// List of document types the extension will run against, supported documents
+// need a scheme, so adding support for 'file' and 'untitled'(in memory)
+const supportedFiles = getSupportedStaticAnalysisFileTypes().map(item => { return { language: item, scheme: 'file' }; });
+const supportedUntitled = getSupportedStaticAnalysisFileTypes().map(item => { return { language: item, scheme: 'untitled' }; });
+const supportedDocuments = [...supportedFiles, ...supportedUntitled];
```

Comment:
The `scheme` is not needed when just using a string instead of an object per https://code.visualstudio.com/api/references/vscode-api#DocumentSelector, so we can just use the return value of `getSupportedStaticAnalysisFileTypes()` directly:

> Note that a document selector that is just a language identifier selects all documents, even those that are not saved on disk.